NetCasbin/CoreEnforcer.cs

```diff
@@ -213,11 +213,13 @@ public async Task LoadPolicyAsync()
         public bool LoadFilteredPolicy(Filter filter)
         {
             model.ClearPolicy();
-            if (!IsFiltered())
+            if (adapter is not IFilteredAdapter filteredAdapter)
```

Comment:
Don't understand why here would throw an exception if adapter is `IFilteredAdapter`.

```C#
!IsFiltered()
```

is the same as

```C#
adapter is not IFilteredAdapter filteredAdapter
```

Reply:
It will throw an exception when the adapter is not IFilteredAdapter. The result of `!IsFiltered()` will also return false when the adapter is `IFilteredAdapter` but `IsFiltered` is true.